setup.py

```diff
@@ -29,6 +29,7 @@
     'ray',
     'scikit-image',
     'scipy',
+    'seaborn',
```

Comment:
Can you add it to the `dev` dependencies instead? It should only be needed when running benchmarking.

Reply:
Fixed